using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -411,7 +471,9 @@ If your application uses more complicated URLs, such as Angular URLs, we recomme
 This method was added in Core version 1.4.0 and Identity version 1.1.0_._
 {% endhint %}
 
-Retrieve Adobe visitor data as a URL query parameter string for consumption in hybrid mobile applications. There is no leading _?_ or _&_ punctuation, as the caller is responsible for placing the string in the correct location of their resulting URL. The following information is added to the string that is returned in the [AdobeCallback](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#adobecallback) instance:
+This `getUrlVariables()` API gets the Visitor ID Service variables in URL query parameter form for consumption in hybrid app. This method will return an appropriately formed string containing Visitor ID Service URL variables. There will be no leading (&) or (?) punctuation, as the caller is responsible for placing it in their resulting java.net.URI in the correct location.
```

Comment:
Reword this section to read as follows:

"This API gets the Visitor ID Service variables in the URL query parameter form, and these variables will be consumed in the hybrid app. This method returns an appropriately formed string that contains the Visitor ID Service URL variables. There will be no leading (&) or (?) punctuation marks because the caller is responsible for placing the punctuation marks in their resulting `java.net.URI` in the correct location.